build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -3,51 +3,16 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
 
   <PropertyGroup>
+    <InitialOutDir>$(OutDir)</InitialOutDir>
     <NuGetOutDir>$(OutDir)Packages\</NuGetOutDir>
     <OutDir>$(OutDir)Raw\</OutDir>
+    <PackCommand>$(DotNetTool) pack $(MSBuildThisFileDirectory)Sdkproject.csproj --no-build -o $(NuGetOutDir) /p:NuspecBasePath=$(InitialOutDir) /p:PackageVersion=$(Version)</PackCommand>
```

Comment:
Oh sorry, I misread the other file. It would have to be something like /p:NuSpecProperties='version=$(Version)' (not sure about quoting/escaping the embedded `=`) to get rid of the target.